libid/fractalb.cpp

```diff
@@ -785,6 +785,7 @@ bool mandel_bf_setup()
     switch (g_fractal_type)
     {
     case fractal_type::MANDELFP:
+    case fractal_type::BURNINGSHIP:
```

Comment:
Not really a fan of this switch getting even bigger as it couples different fractal types together.  But I guess it's OK for now.

Reply:
Nothing changed here.